docs/src/modules/java/pages/workflows.adoc

```diff
@@ -134,6 +134,23 @@ IMPORTANT: We are returning the internal state directly back to the requester. I
 
 A full transfer workflow source code sample can be downloaded as a link:../java/_attachments/workflow-quickstart.zip[zip file]. Follow the `README` file to run and test it.
 
+== Deleting state
+
+If you want to delete the workflow state, you can use the `effect().delete` method. This will remove the workflow from the system.
```

Comment:
```suggestion
If you want to delete the workflow state, you can use the `effects().delete` method. This will remove the workflow from the system.
```